scene/resources/material.cpp

```diff
@@ -780,7 +780,9 @@ void BaseMaterial3D::_update_shader() {
 	if (flags[FLAG_DISABLE_FOG]) {
 		code += ",fog_disabled";
 	}
-
+	if (flags[FLAG_MOTION_DRAW_DISABLED]) {
+		code += ",motion_draw_disabled";
+	}
```

Comment:
```suggestion
	}

```
Minor nit, for readability I'd keep the space from before grouping the flags